packages/easy-coding-standard/src/Console/Command/MarkdownCodeFormatterCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\Console\Command;
+
+use ReflectionProperty;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symplify\EasyCodingStandard\Configuration\Configuration;
+use Symplify\EasyCodingStandard\Configuration\Exception\NoMarkdownFileException;
+use Symplify\EasyCodingStandard\Console\Style\EasyCodingStandardStyle;
+use Symplify\EasyCodingStandard\FixerRunner\Application\FixerFileProcessor;
+use Symplify\PackageBuilder\Console\ShellCode;
+use Symplify\SmartFileSystem\SmartFileInfo;
+use Symplify\SmartFileSystem\SmartFileSystem;
+
+final class MarkdownCodeFormatterCommand extends Command
+{
+    /**
+     * @var SmartFileSystem
+     */
+    private $smartFileSystem;
+
+    /**
+     * @var FixerFileProcessor
+     */
+    private $fixerFileProcessor;
+
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @var EasyCodingStandardStyle
+     */
+    private $easyCodingStandardStyle;
+
+    public function __construct(
+        SmartFileSystem $smartFileSystem,
+        FixerFileProcessor $fixerFileProcessor,
```

Comment:
The `FixerFileProcessor` handles only php-cs-fixer.

We also need to add the `SniffFileProcessor` to process the file, it handles the PHP_CodeSniffer.

Possibly in 2nd iteration.

Reply:
I've added `SniffFileProcessor` ->processFile() after `FixerFileProcessor`->processFile()